modules/client/test/integration/addresslistVoting-client/methods.test.ts

```diff
@@ -281,6 +299,70 @@ describe("Client Address List", () => {
     });
   });
 
+  describe("Plugin installation", () => {
+    it("Should prepare the installation of a token voting plugin", async () => {
+      const networkSpy = jest.spyOn(JsonRpcProvider, "getNetwork");
+      networkSpy.mockReturnValueOnce({
+        name: "goerli",
+        chainId: 31337,
```

Comment:
the chain ID of goerli is 5

Reply:
I know but if I change the value of the chain the code complains because i changed it and it also detects that it has ens when it's not true and throw some errors